gdpr_cookies.module

```diff
@@ -84,31 +88,31 @@ function gdpr_cookies_module_implements_alter(&$implementations, $hook) {
 }
 
 /**
- * Implements hook_page_attachments().
+ * Implements hook_library_info().
  */
-function gdpr_cookies_library() {
-  $libraries['gdpr-cookies'] = array(
-    'title' => 'GDPR Cookies',
-    'version' => '1.0',
+function gdpr_cookies_library_info() {
+  $libraries['tarteaucitron'] = array(
+    'title' => 'Tarte au Citron',
+    'version' => '1.20.1',
     'js' => array(
       backdrop_get_path('module', 'gdpr_cookies') . '/js/tarteaucitron/tarteaucitron.js' => array(
         'type' => 'file',
-        'scope' => 'footer',
+        'scope' => 'header',
```

Comment:
I don't know if it's possible to put this in the footer, but if so it will make the page load more quickly. (I'm assuming it's not, or you would have left it there)

Reply:
This was another attempt to fix the library; I will test back in the footer and remove if it can be.